neo/rawio/blackrockrawio.py

```diff
@@ -632,12 +677,14 @@ def _get_spike_raw_waveforms(self, block_index, seg_index, unit_index, t_start,
         return waveforms
 
     def _event_count(self, block_index, seg_index, event_channel_index):
+        # TODO: Should this only return NonNeural events or also Comment events ?
```

Comment:
currently only NonNeural events are counted. Is there a reason to exclude Comment events here?

Reply:
I don't remember.
I guess here both nonneural + comments should count in different event channel.